src/zjs_util.h

```diff
@@ -97,4 +97,18 @@ void zjs_default_convert_pin(uint32_t orig, int *dev, int *pin);
 uint16_t zjs_compress_32_to_16(uint32_t num);
 uint32_t zjs_uncompress_16_to_32(uint16_t num);
 
+/**
+ * Serialize a jerry_value_t into a buffer. The returned buffer must be freed
+ * after use.
+ *
+ * @param value     Value to serialize. This can be a Buffer, Array (of bytes)
+ *                  or String.
+ * @param maxLen    Maximum length of data to serialize. If the length does not
+ *                  matter you can specify 0 or NULL.
+ * @param actualLen Actual length of buffer
```

Comment:
You really mean this is an output with the bytes written. Maybe you should also allow that to be NULL in case they don't care? E.g. "If not NULL, receives the number of bytes written to the buffer."